src/main/java/org/sagebionetworks/web/client/presenter/EntityPresenter.java

```diff
@@ -352,10 +352,7 @@ public void onError(Throwable caught) {
     headerWidget.configure();
     if (caught instanceof NotFoundException) {
       show404();
-    } else if (
-      caught instanceof ForbiddenException &&
-      authenticationController.isLoggedIn()
```

Comment:
Previously, if anonymous it would call StuAlert handleException() which passed in null for the entity ID (and version).  Now, we always call show403(entityId, versionNumber), but only show the open invitations if you are logged in (see show403() in this class).